### DISCUSSION TITLE
useRouter in Kombination mit useStore funktioniert nicht

### DISCUSSION THREAD

**Discussion by therealchrismcfly** [neuefische/questions#161] [Web] [answered]
Hallo zusammen,

ich hänge an einem Problem fest und versuche jetzt mal hier mein Glück: 

**Hintergrundinformationen:**
Der Plan ist, dass man über den + Button Projekte zu einer Liste hinzufügt. Klickt man auf den Edit Button eines einzelnen Projekts, bekommt man die entsprechenden Informationen angezeigt. Die projects kommen aus dem useStore. 

So sieht die Editieren Seite aus: 
<img width="783" alt="Bildschirmfoto 2022-09-22 um 13 33 12" src="https://user-images.githubusercontent.com/107792731/191736067-1f38d9fb-f017-4f70-a985-4e498e709ddd.png">

**Das Problem:**
Klicke ich auf edit, wird die URL korrekt angezeigt, die Seite ist jedoch leer. Wenn ich projects logge, werden die Daten aus dem useStore angezeigt, wenn ich entry logge bekomme ich ein undefined. Es hakt also irgendwie daran, auf ein einzelnes project - Objekt zuzugreifen, aber ich hänge fest, was hier die Lösung sein könnte.

Hier der Link zum Repo:
https://github.com/therealchrismcfly/diy-project-planner/pulls

Vielen Dank schonmal für eure Hilfe & liebe Grüße 

Chris :-)

**Comment by mbosselmann** [comment, marked as answer]
Hallo Chris,
wenn du in Zeile 22 in der `find` Methode nicht die id in eine Nummer umwandelst, sondern als String behältst, sollte es gehen. :)
Ich hoffe, dass dir das schon weiterhilft!

**Reply by therealchrismcfly** [reply]
Es funktioniert!!! 🥳 Vielen vielen vielen Dank dir!
